src/redux/products/products.reducer.js

```diff
@@ -25,12 +25,14 @@ export const initialState = {
   isHotItem: true,
   sortByRate: 0,
   sortByPopularity: -1,
-  colorsFilter: [],
-  patternsFilter: [],
-  categoryFilter: undefined,
-  priceFilter: [0, 99999],
-  searchFilter: '',
-  isHotItemFilter: false,
+  filters: {
+    colorsFilter: [],
+    patternsFilter: [],
+    categoryFilter: null,
+    priceFilter: [0, 99999],
+    searchFilter: '',
+    isHotItemFilter: false
+  },
   filtredProducts: [],
```

Comment:
We should have only one array for the products. Please, get rid of the `filtredProducts`, use only one array and put there filtered products returned from the backend

Reply:
Now this is one products field

Reply:
fixed